src/Pkg.jl

```diff
@@ -273,6 +270,41 @@ const develop = API.develop
 #TODO: Will probably be deprecated for something in PkgDev
 const generate = API.generate
 
+"""
+    Pkg.dependencies()::Dict{UUID, PackageInfo}
```

Comment:
Should this include things only in the Project, or also in the Manifest or should that be an option?